rules/cross-platform/command_and_control_google_drive_malicious_file_download.toml

```diff
@@ -0,0 +1,86 @@
+[metadata]
+creation_date = "2023/06/19"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/06/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies potential malicious file download and execution from Google Drive. The rule checks for download activity from
+Google Drive URL, followed by the creation of files commonly leveraged by or for malware. This could indicate an attempt
+to run malicious scripts, executables or payloads.
+"""
+false_positives = [
+    "Approved third-party applications that use Google Drive download URLs.",
+    "Legitimate publicly shared files from Google Drive."
+]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Malicious File Downloaded from Google Drive"
+note = """## Setup
```

Comment:
we can probably remove this now that min stack > 8.2 (outside of the scope of this PR)

Reply:
Removed, thanks for the reminder!